gem/static/js/springster.js

```diff
@@ -30,13 +30,20 @@
   var stickyHeader = function() {
     var header = document.getElementById("header-wrapper");
     var content = document.getElementById("content-wrapper");
+    var menuList = document.getElementById("nav-list");
 
     var onResizing = function(event) {
-      if (window.innerWidth < 1024 ){
+      if (window.innerWidth < 1024){
         content.style.backgroundColor =  "#7300ff";
       } else {
         content.style.backgroundColor =  "transparent";
       }
+      
```

Comment:
this is just for the menu items currently, so when the menu wraps the last items, js pics up the altered height of the container, and gives it a smaller width to force down another item

_Homepage navigation - Can we force a second link down to the second line when it breaks. On e.g. the Spanish example the link Mi Vida is the only item on the second line_